quirks/password-rules.json

```diff
@@ -189,6 +189,9 @@
     "ssa.gov": {
         "password-rules": "required: lower; required: upper; required: digit; required: [~!@#$%^&*];"
     },
+    "sunlife.com": {
+        "password-rules": "minlength: 8; maxlength: 10; required: digit; allowed: upper, lower;"
```

Comment:
The website’s rules you quote say the site must have at least one letter, but this only says that letters are allowed. I think this should be:

```
minlength: 8; maxlength: 10; required: digit; required: upper, lower;
```

Reply:
Thanks, I've fixed this now!